doc/converse/manual.rst

```diff
@@ -1459,77 +1459,6 @@ Returns the number of children of nodeNum in the spanning tree.
 This function fills the array children with node numbers of children of
 nodeNum in the spanning tree.
 
-Isomalloc
```

Comment:
Why are we removing this entirely? Shouldn't it be updated?

Reply:
Readded the documentation and updated it based on the changes in the PR.